sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTestBase.java

```diff
@@ -81,14 +87,15 @@ protected String getTestName() {
     }
 
     void beforeTestSetup() {
+        KeyVaultCredentialPolicy.clearCache();
```

Comment:
We don't need this for the admin client Tests?

Reply:
Yes, we do. Let me make sure it's there.

Reply:
We set it on `KeyVaultAdministrationClientTestBase`.